scripts/test.sh

```diff
@@ -13,7 +13,7 @@ echo "mode: count" > profile.cov
 # Standard go tooling behavior is to ignore dirs with leading underscors
 for dir in $(find . -maxdepth 3 -not -path './.git*' -not -path '*/_*' -not -path '_*' -type d -not -path './examples*' -not -path './vendor*' -not -path './output*' -not -path './dist*' -not -path './build*' -not -path './bin*' -not -path './scripts*' -not -path './pact-*'); do
   if ls $dir/*.go &> /dev/null; then
-    go test -covermode=count -coverprofile=$dir/profile.tmp $dir
+    go test  -covermode=count -coverprofile=$dir/profile.tmp $dir
```

Comment:
Any reason for the extra space?

Reply:
oops, added a `-v` for local testing to so i could see the which tests were actually failing since the log lines aren't helpful (mock server never actually starts). will fix!